config.js

```diff
@@ -8,6 +8,14 @@ var api = {
   port: process.env.API_PORT_8080_TCP_POST || process.env.API_PORT || 8080
 };
 
+var oauth = {
+  oauthUrl: process.env.OAUTH_URL,
+  clientId: process.env.CLIENT_ID,
+  clientSecret: process.env.CLIENT_SECRET,
+  username: process.env.KEYCLOAK_U,
+  password: process.env.KEYCLOAK_P
+};
+
```

Comment:
Could we lose the separate object for this and shove it straight in module.exports? We only have a dedicated api object because we have a hack for the mock proxy going on.